src/components/Input/Input.js

```diff
@@ -138,7 +153,33 @@ class Input extends React.Component {
     }
   };
 
-  render() {
+  renderPasswordField = () => {
+    const toggleState = () => this.state._type === 'password' ? 'text' : 'password';
+    const handleClick = (evt) => {
+      evt.preventDefault();
+      this.setState({
+        _type: toggleState(this.state._type)
+      });
+    }
+
+    const getText = () => {
+      return this.state._type === 'password' ? 'Show' : 'Hide'
+    }
+
+    return (
+      <div style={{position:'relative'}}>
+          <div style={{position:'absolute', right: '10px', top: '25%'}}>
```

Comment:
Didn't feel like using styled-components for these? I can certainly see why, just curious.

Reply:
This was honestly just carelessness on my part leaving this in while I played around with it. Changed to styled-components